addons/account/models/chart_template.py

```diff
@@ -1020,6 +1020,7 @@ class AccountReconcileModelTemplate(models.Model):
         ('percentage', 'Percentage of balance')
         ], required=True, default='percentage')
     amount = fields.Float(digits=0, required=True, default=100.0, help="Fixed amount will count as a debit if it is negative, as a credit if it is positive.")
+    tax_include = fields.Boolean(string='Tax Include')
```

Comment:
use the same label as price_include field in account.tax => 'Tax Included in Price'

Reply:
and rename into something more explicit
is_tax_included